core/bin/external_node/src/config/mod.rs

```diff
@@ -770,7 +770,7 @@ impl OptionalENConfig {
     }
 
     const fn default_protective_reads_persistence_enabled() -> bool {
```

Comment:
Nit: You can remove this function altogether and use `#[serde(default)]`. Also, remove "By default, set to `true`..." as well.

Reply:
Updated, also changed default not only for EN but in other places as well